src/main/java/com/nomiceu/nomilabs/gregtech/metatileentity/multiblock/MetaTileEntityCreativeTankProvider.java

```diff
@@ -101,6 +100,6 @@ public void addInformation(ItemStack stack, @Nullable World world, @NotNull List
 
     @Override
     public boolean shouldShowInJEI() {
-        return LabsModeHelper.isNormal();
+        return false;
```

Comment:
Isn't the tank provider completely useless in hm ? This just hides it from JEI, we could maybe totally get rid of it ? Same thing for the other nm-only multis from labs